templates/config.j2

```diff
@@ -53,7 +54,9 @@ providers:
     watch: true
     exposedByDefault: {{ traefik_docker_exposed_by_default | lower }}
     useBindPortIP: {{ traefik_docker_bind_port_ip | lower }}
+    {% if traefik_docker_network_name is defined and traefik_docker_network_name %}
```

Comment:
The whole template is built without the lstrip_blocks magic, please just keep it like that and put the condition to the beginning of the line.